frontend/src/pages/TermsOfUse/TermsOfUse.tsx

```diff
@@ -87,7 +60,7 @@ export const TermsOfUse: React.FC = () => {
         to have access to the organization’s data. View-only users can only view
         data provided to or collected by Crossfeed.
       </p>
-      {getMaximumRole(user) === 'admin' && (
+      {touVersion === 'admin' && (
```

Comment:
I don't think this will work. `touVersion` may be equal to `v1-admin` but never equal to `admin`.

Reply:
thanks, meant to be maximumRole not touVersion